src/app/CommandHandler.h

```diff
@@ -171,6 +199,14 @@ class CommandHandler : public Messaging::ExchangeDelegate
     void OnInvokeCommandRequest(Messaging::ExchangeContext * ec, const PayloadHeader & payloadHeader,
                                 System::PacketBufferHandle && payload, bool isTimedInvoke);
 
+    /**
+     * Checks that all CommandDataIB within InvokeRequests is correct as per spec.
```

Comment:
```suggestion
     * Checks that all CommandDataIB within InvokeRequests satisfy the spec's general constraints for CommandDataIB
```
In particular, this is not checking anything about the actual command data in there...